### PR TITLE
release for xpd again

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: recipes-tag/regolith/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "regolith" %}
-{% set version = "0.2.0" %}
-{% set sha256 = "d278f44373d27abc858ded0e6f06b7dc35189dbe737f87025703b0982ae10a20" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "3b401670ed958ce2bb63fd3584ff472baae8bda1f00d2689cc1091adaf0ca1b5" %}
```
Comment (mrakitin) [inline recipes-tag/regolith/meta.yaml]: Is this package needed at all? I don't see any dependency on it in this PR.

Comment (CJ-Wright) [reply recipes-tag/regolith/meta.yaml]: I could remove the recipe completely

**Comment by CJ-Wright** [comment]
@mrakitin ready for another round!

**Comment by CJ-Wright** [comment]
Actually do you mind if I self merge?
This way I can run the deployment first thing tomorrow.

**Comment by mrakitin** [comment]
Sorry, was too late last night to check the status.

**Pull request merged**